core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -284,6 +266,32 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
         // this.store.updateEditorState(EDITOR_STATE.LOADING);
     }
 
+    /**
+     * Handles the reload of content in the editor.
+     * If the editor state is LOADED and the content is not VTL, it reloads the iframe.
+     * If the content is VTL, it loads the VTL iframe content.
+     * @memberof EditEmaEditorComponent
+     */
+    handleReloadContent() {
+        this.store.stateLoad$
+            .pipe(
+                takeUntil(this.destroy$),
+                filter((editorState) => editorState === EDITOR_STATE.LOADED),
+                switchMap(() => this.store.code$)
+            )
```

Comment:
Write a selector that gives you exactly what you need, don't merge selectors in the components.

Reply:
I tried to do this, but that selector is updated more times than desired, I imagine due to a side effect from other updaters

https://github.com/dotCMS/core/pull/27922#issuecomment-1989259090

Reply:
Well... that's bad, we need to find out why.